pepper-apis/src/main/java/org/broadinstitute/ddp/security/JWTConverter.java

```diff
@@ -30,6 +38,8 @@
     private static final Logger LOG = LoggerFactory.getLogger(JWTConverter.class);
     private static final String DEFAULT_ISO_LANGUAGE_CODE = "en";
     private Map<String, JwkProvider> jwkProviderMap = new HashMap<>(); // Map of Auth0ClientIds -> jwkProviders
+    private Cache<String, DDPAuth> jwtToDDPAuthCache;
+    private Cache<Long, Set<String>> userIdToJwtCache;
 
```

Comment:
This is a special case where we need to use to caches: one to map the stuff we actually care about and the other to keep a mapping from user id to the entries so we can look them up an evict them as we get model modifications.